src/notify/app/validators/request_validator.py

```diff
@@ -21,6 +21,15 @@ def verify_headers(headers: dict, api_key: str) -> tuple[bool, str]:
 
     return True, ""
 
+def verify_batch_headers(headers: dict) -> tuple[bool, str]:
+    lc_headers = header_keys_to_lower(headers)
+    if lc_headers.get('authorization') is None:
+        return False, "Authorization header not present"
```

Comment:
I've preserved the original phrasing but it is a bit different to the other failures, e.g. `Missing API key header`, maybe we want to standardise?